bedrock/firefox/templates/firefox/ios.html

```diff
@@ -93,6 +104,8 @@
       <li class="sync-history">{{ _('History') }}</li>
       <li class="sync-passwords">{{ _('Passwords') }}</li>
       <li class="sync-tabs">{{ _('Open tabs') }}</li>
+      <li class="sync-bookmarks">{{ _('Bookmarks') }}</li>
+      <li class="sync-reading-list">{{ _('Reading List') }}</li>
```

Comment:
Even if we never shipped, the page is localized and live for several language. We need new strings behind a flag.